command/agent_test.go

```diff
@@ -2759,7 +2743,7 @@ cache {}
 	defer os.Remove(configPath)
 
 	// Start the agent
-	_, cmd := testAgentCommand(t, logger)
+	_, cmd := testAgentCommand(t, nil)
```

Comment:
The Run method we invoke later creates its own logger, so no point in passing one in here.